README.md

```diff
@@ -345,6 +346,16 @@ To add a resource or contribute, please check out our contribution guide [here](
 
 ---
 
+## Ionic
+
+- [Getting Started with Ionic: Angular Concepts & Syntax](https://www.joshmorony.com/ionic-2-first-look-series-new-angular-2-concepts-syntax/)
+- [Simple animation library to use in Ionic Apps](https://www.youtube.com/watch?v=zCwWKhiED6E)
+- [An Introduction to HTTP Requests & Fetching Data in Ionic](https://www.joshmorony.com/an-introduction-to-http-requests-fetching-data-in-ionic/)
```

Comment:
Please remove this link

Reply:
ok! i will, one question tho, can i add a repo that provides a curated list of ionic liibraries?

Reply:
> Please remove this link

It's done, do check my newest commit. Thanks